ibis/backends/dask/tests/test_import.py

```diff
@@ -0,0 +1,5 @@
+# Trivial test to make sure CI runs and package is importable
```

Comment:
can you show me on the CI where this ran?

Reply:
Oh great catch. Looks like `run_tests.sh` still expects the directory structure to be `ibis/$BACKEND/tests` (see https://github.com/ibis-project/ibis/blob/master/ci/run_tests.sh#L8), where I think given the new directory structure this should be ` ibis/backends/$BACKEND/tests`. 

This is a problem in the regular tests as well: https://github.com/ibis-project/ibis/pull/2536/checks?check_run_id=1433682341#step:4:6 if I understand this correctly.

Reply:
oh! yes pls split out to a separate PR if you would.

Reply:
https://github.com/ibis-project/ibis/pull/2538

Reply:
I've confirmed CI is now running: 

```
ibis/backends/dask/tests/test_import.py .                                [100%]
```